Formula/jetty-runner.rb

```diff
@@ -17,6 +17,8 @@ def install
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/jetty-runner --version", 1)
```

Comment:
And since it hasn't been said before: thanks for your contribution to Homebrew! 🎈